src/components/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">0</div>
+    <button className="button gray" type="button">AC</button>
+    <button className="button gray" type="button">+/-</button>
+    <button className="button gray" type="button">%</button>
+    <button className="button orange" type="button">÷</button>
+    <button className="button" type="button">7</button>
+    <button className="button" type="button">8</button>
+    <button className="button" type="button">9</button>
+    <button className="button orange" type="button">×</button>
+    <button className="button" type="button">4</button>
+    <button className="button" type="button">5</button>
+    <button className="button" type="button">6</button>
+    <button className="button orange" type="button">-</button>
+    <button className="button" type="button">1</button>
+    <button className="button" type="button">2</button>
+    <button className="button" type="button">3</button>
+    <button className="button orange" type="button">+</button>
+    <button className="button zero" type="button">0</button>
+    <button className="button" type="button">.</button>
+    <button className="button orange" type="button">=</button>
+  </div>
+);
```

Comment:
- [ ] Ouch! You're almost there! However, according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md), it's necessary to create at least one child component and incorporate it within your Calculator component. To meet this requirement, kindly consider extracting a portion of the code from this component and placing it in a separate component. Then, utilize that newly created component as the child component here. This adjustment will ensure compliance with the project guidelines. Keep up the good work!

Reply:
Thank you for the good sighting 🥂 .